src/Package.js

```diff
@@ -49,7 +49,6 @@ export default class Package {
   get allDependencies() {
     return objectAssign(
       {},
-      this.peerDependencies,
```

Comment:
I'm tempted to say that we should leave `allDependencies` as is and have a `bootstrapDependencies` or filter out peerDeps. Maybe that's optimizing for the future too much though...

Actually... let's just do this, I don't think it'll cause any problems and if we need it in the future it's an easy refactor.